lighthouse-plugin-publisher-ads/audits/async-ad-tags.js

```diff
@@ -79,10 +79,11 @@ class AsyncAdTags extends Audit {
       return auditNotApplicable.NoTag;
     }
 
-    const numAsync = array.count(tagReqs, isAsync);
-    const numTags = tagReqs.length;
+    const numSync = array.count(tagReqs, isAsync) - tagReqs.length;
```

Comment:
Nit: Shouldn't it be tagReqs.length - array.count(tagReqs, isAsync)?

Reply:
Oops, will send out a fix!